Python/Product/PythonTools/PythonTools/Repl/PythonDebugReplEvaluator.cs

```diff
@@ -203,7 +202,8 @@ internal class PythonDebugReplEvaluator :
             if (_activeEvaluator != null) {
                 return _activeEvaluator.FormatClipboard();
             }
-            return String.Empty;
+            //Formatting for experimental debugger
```

Comment:
I don't think that comment is necessarily true. If it's `null` it could also be that there is no debugging happening currently, regardless of which debugger is selected in tools/options.

`CustomDebugAdapterProtocolExtension.CanUseExperimental()` lets you know if experimental debugger is currently debugging.

There's nothing debugger specific about paste, so we could unconditionally call `PythonCommonInteractiveEvaluator.FormatClipBoard`and take out the previous `if` statement ie not call `_activateEvaluator.FormatClipboard`.

Reply:
I agree with removing the comment. 

I considered changing "FormatClipboard()" to static function in PythonCommonInteractiveEvaluator.cs and calling that function directly through SelectableReplEvaluator.cs (line 356) and PythonDebugReplEvaluator.cs (line 203) through their respective CurrentWindows. However, there may be differences in the "CurrentWindow" property in PythonDebugReplEvaluator.cs and PythonCommonInteractiveEvaluator.cs. If we start interchanging them (using PythonCommonInteractiveEvaluator.cs property everywhere except FormatClipboard()) within the instance object, we could potentially end up with unexpected problems. 

I'm forced to use CurrentWindow property from PythonDebugReplEvaluator.cs for experimental debugger, but I want to avoid doing it for sections of code which don't strictly require it. And I don't want to fix something that is not broken :)

Reply:
It's going to be the same Window from both places. `PythonDebugReplEvaluator` passes its `CurrentWindow` to `PythonDebugProcessReplEvaluator` (which derives from `PythonCommonInteractiveEvaluator`).

`PythonDebugReplEvaluator` will get a cleanup when we remove legacy debugger. We can remove that `if` now or later when all the other `if`s go away, doesn't matter much.

Reply:
Modified. 

I've kept the FormatClipboard() in PythonCommonInteractiveEvaluator.cs because it implements IInteractiveEvaluator which requires FormatClipboard. This is being called by SelectableReplEvaluator.cs.